src/xero/exceptions.py

```diff
@@ -50,10 +50,16 @@ def __init__(self, response):
 
         elif response.headers["content-type"].startswith("text/html"):
             payload = parse_qs(response.text)
-            self.errors = [payload["oauth_problem"][0]]
-            self.problem = self.errors[0]
-            super().__init__(response, payload["oauth_problem_advice"][0])
-
+            if payload:
```

Comment:
Handle the undocumented format of error messages from xero when sending malformed UUIDs for a list filter request.